tools/code_review_tools/treeherder.py

```diff
@@ -5,15 +5,23 @@
 import slugid
 from thclient import TreeherderClient
 
+JOBS_URL = "https://treeherder.mozilla.org/#/jobs"
 
-def get_job_url(task_id, run_id, **params):
+
+def get_job_url(repository, revision, task_id=None, run_id=None, **params):
     """Build a Treeherder job url for a given Taskcluster task"""
-    treeherder_client = TreeherderClient()
-    uuid = slugid.decode(task_id)
+    assert isinstance(repository, str) and repository, "Missing repository"
+    assert isinstance(revision, str) and revision, "Missing revision"
+
+    params.update({"repo": repository, "revision": revision})
```

Comment:
Nit: assert that `repo` and `revision` are not in params

Reply:
Done